static/css/student/profile.css

```diff
@@ -452,4 +452,88 @@ footer h6 {
 		width: 195px;
 		border-radius: 30px;
 	}
+}
+
+.course-card {
+	border-radius: 5px;
+	color: rgb(34,34,34);
+
+	margin: 10px;
+	height: 80px;
+}
+.course-card p{
```

Comment:
you can combine these two into:
.course-card {
   border-radius:...
   color: ....
   ....
   p {
         font-size:...
         .....
   }
}

Reply:
sorry the spacing wasn't clear but is should be:
.course-card {
	border-radius: 5px;
	color: rgb(34,34,34);

 	margin: 10px;
	height: 80px;
        p {
         .....
        }
}